ledger/financial/cmd/client/batcher.go

```diff
@@ -0,0 +1,226 @@
+package main
+
+import (
+	"bytes"
+	"crypto/rand"
+	"crypto/sha512"
+	"encoding/base64"
+	"encoding/hex"
+	"fmt"
+	"io/ioutil"
+	"ledger/financial/handler"
+	"ledger/financial/payload"
+	"ledger/financial/state"
+	"net/http"
+	"net/url"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/hyperledger/sawtooth-sdk-go/protobuf/batch_pb2"
+	"github.com/hyperledger/sawtooth-sdk-go/signing"
+	"google.golang.org/protobuf/proto"
+
+	tx_pb2 "github.com/hyperledger/sawtooth-sdk-go/protobuf/transaction_pb2"
+)
+
+// batcher defines the primitive to send transactions
+type batcher interface {
+	sendTransaction(payload payload.FinancialPayload) (string, error)
+}
+
+// newBatcher returns a new initialized batcher. url must be the REST endpoint,
+// and key must by the filepath to a private key that is allowed to send
```

Comment:
by or be ?

Reply:
to by or not to be ? 
this is "be", indeed :D